bundles/org.openhab.ui/web/src/pages/settings/rules/scene/scene-edit.vue

```diff
@@ -0,0 +1,613 @@
+        this.$set(this, 'selectedItems', this.selectedItems.filter((i) => i !== itemName))
+        this.buildActionModules()
+      })
+    },
+    selectItems (items) {
+      console.log(items)
+      this.$set(this, 'selectedItems', items)
+      this.buildActionModules()
+    },
+    reorderModule (ev, section) {
+      const newSection = [...this.rule[section]]
+      newSection.splice(ev.to, 0, newSection.splice(ev.from, 1)[0])
+      this.$set(this.rule, section, newSection)
+    },
+    buildActionModules () {
+      const currentItemNames = this.rule.actions.map((a) => a.configuration.itemName)
+      const modulesToRemove = this.rule.actions.filter((a) => this.selectedItems.indexOf(a.configuration.itemName) < 0)
+      if (modulesToRemove.length > 0) console.debug('Removing: ' + modulesToRemove.map((m) => m.configuration.itemName).join(', '))
+      this.$set(this.rule, 'actions', this.rule.actions.filter((a) => this.selectedItems.indexOf(a.configuration.itemName) >= 0))
+      const itemsToAdd = this.selectedItems.filter((i) => !this.rule.actions.some((a) => a.configuration.itemName === i))
+      if (itemsToAdd.length > 0) console.debug('Adding: ' + itemsToAdd.join(', '))
+
+      let moduleId = 1
+      itemsToAdd.forEach((itemName) => {
+        for (; ['triggers', 'actions', 'conditions'].some((s) => this.rule[s].some((m) => m.id === moduleId.toString())); moduleId++);
+        console.debug('new moduleId=' + moduleId)
+        const newModule = {
+          id: moduleId.toString(),
+          configuration: {
+            itemName,
+            command: null
+          },
+          type: 'core.ItemCommandAction'
+        }
+        this.rule.actions.push(newModule)
+      })
+      const statePromises = itemsToAdd.map((itemName) => this.$oh.api.getPlain('/rest/items/' + itemName + '/state'))
+      Promise.all(statePromises).then((states) => {
+        states.forEach((state, idx) => {
+          const module = this.rule.actions.find((a) => a.configuration.itemName === itemsToAdd[idx])
+          this.$set(module.configuration, 'command', state)
+        })
+      })
+    },
+    updateCommandFromCurrentState (ev, module) {
+      if (ev) ev.cancelBubble = true
+      const itemName = module.configuration.itemName
+      this.$oh.api.getPlain('/rest/items/' + itemName + '/state').then((state) => {
+        this.$set(module.configuration, 'command', state)
+        // this.$f7.toast.create({
+        //   text: `Updated desired state of ${itemName} to ${state}`,
+        //   destroyOnClose: true,
+        //   closeTimeout: 2000
+        // }).open()
+      })
+    },
+    testCommand (ev, module) {
+      if (ev) ev.cancelBubble = true
+      const itemName = module.configuration.itemName
+      const command = module.configuration.command
+      this.$oh.api.postPlain('/rest/items/' + itemName, command, 'text/plain', 'text/plain').then((state) => {
+        // this.$f7.toast.create({
```

Comment:
IMO a toast notification on commands send is always useful, so I'd vote for displaying a toast notification here.
I like to get some feedback in the UI when I interact with physical equipment, even though I am usually besides my equipment when creating scenes.

Reply:
You're right, I was on the fence on these toasts and finally decided against, but the one for commands still makes sense. I have removed the commented code for the "current state to module" method and re-enabled this one.